api/docs/tough-cookie.parsedate.md

```diff
@@ -0,0 +1,122 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [tough-cookie](./tough-cookie.md) &gt; [parseDate](./tough-cookie.parsedate.md)
+
+## parseDate() function
+
+Parse a cookie date string into a [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)<!-- -->. Parses according to [RFC6265 - Section 5.1.1](https://www.rfc-editor.org/rfc/rfc6265.html#section-5.1.1)<!-- -->, not [Date.parse()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse)<!-- -->.
+
+**Signature:**
+
+```typescript
+export declare function parseDate(cookieDate: Nullable<string>): Date | undefined;
+```
+
+## Parameters
+
+<table><thead><tr><th>
+
+Parameter
+
+
+</th><th>
+
+Type
+
+
+</th><th>
+
+Description
+
+
+</th></tr></thead>
+<tbody><tr><td>
+
+cookieDate
+
+
+</td><td>
+
+Nullable&lt;string&gt;
+
+
+</td><td>
+
+the cookie date string
+
+
+</td></tr>
+</tbody></table>
+**Returns:**
+
+Date \| undefined
+
+## Remarks
+
+\#\#\# RFC6265 - 5.1.1. Dates
```

Comment:
This isn't rendered properly 🫤

Reply:
I'll file an issue about the generated content against `@microsoft/api-documenter` if one doesn't already exist.